Fri Nov 21 11:30:05 CLST 2014 : Apache Derby Network Server - 10.10.1.3 - (1557168) started and ready to accept connections on port 1527
----------------------------------------------------------------
Fri Nov 21 11:30:05 CLST 2014:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.10.1.3 - (1557168): instance a816c00e-0149-d2c2-77d0-00003a42fba7 
on database directory C:\Users\Rafael\Documents\GitHub\pometodos\DB\PokemonUsuarios with class loader sun.misc.Launcher$AppClassLoader@1d44bcfa 
Loaded from file:/C:/Program%20Files/Java/jdk1.8.0_25/db/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_25-b18
user.dir=C:\Program Files\Java\jdk1.8.0_25\db
os.name=Windows 8.1
os.arch=amd64
os.version=6.3
derby.system.home=C:\Users\Rafael\Documents\GitHub\pometodos\DB
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Nov 21 11:30:11 CLST 2014:
Shutting down instance a816c00e-0149-d2c2-77d0-00003a42fba7 on database directory C:\Users\Rafael\Documents\GitHub\pometodos\DB\PokemonUsuarios with class loader sun.misc.Launcher$AppClassLoader@1d44bcfa 
Fri Nov 21 11:30:11 CLST 2014 Thread[DRDAConnThread_2,5,main] (DATABASE = PokemonUsuarios), (DRDAID = {2}), Database 'PokemonUsuarios' shutdown.
----------------------------------------------------------------
Fri Nov 21 11:30:15 CLST 2014:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.10.1.3 - (1557168): instance 560740aa-0149-d2c2-77d0-00003a42fba7 
on database directory C:\Users\Rafael\Documents\GitHub\pometodos\DB\PokemonUsuarios with class loader sun.misc.Launcher$AppClassLoader@1d44bcfa 
Loaded from file:/C:/Program%20Files/Java/jdk1.8.0_25/db/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_25-b18
user.dir=C:\Program Files\Java\jdk1.8.0_25\db
os.name=Windows 8.1
os.arch=amd64
os.version=6.3
derby.system.home=C:\Users\Rafael\Documents\GitHub\pometodos\DB
Database Class Loader started - derby.database.classpath=''
Fri Nov 21 11:35:13 CLST 2014 Thread[DRDAConnThread_3,5,main] Cleanup action starting
java.sql.SQLException: Database 'DBUsuarios' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
Caused by: java.sql.SQLException: Database 'DBUsuarios' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 19 more
============= begin nested exception, level (1) ===========
java.sql.SQLException: Database 'DBUsuarios' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
============= end nested exception, level (1) ===========
Cleanup action completed
Fri Nov 21 11:35:13 CLST 2014 Thread[DRDAConnThread_3,5,main] (DATABASE = DBUsuarios), (DRDAID = {4}), Database 'DBUsuarios' not found.
